dklearn/wrappers.py

```diff
@@ -0,0 +1,256 @@
+from __future__ import division, print_function, absolute_import
+
+from functools import partial
+
+import numpy as np
+import dask
+import dask.array as da
+import dask.bag as db
+from dask import threaded
+from dask.base import Base
+from dask.delayed import delayed, Delayed
+from sklearn.base import clone, is_classifier
+from sklearn.exceptions import NotFittedError
+from sklearn.utils.metaestimators import if_delegate_has_method
+
+from .core import (LazyDaskEstimator, check_X_y, as_lists_of_delayed,
+                   is_list_of)
+from .utils import copy_to
+
+
+__all__ = ('DelayedEstimator', 'Averaged', 'Chained')
+
+
+def _finalize_estimator(res, base):
+    base = clone(base)
+    if hasattr(base, '_update_estimator'):
+        return base._update_estimator(res[0])
+    return copy_to(res[0], base)
+
+
+class DelayedEstimator(Base):
+    """A delayed estimator"""
+    __slots__ = ('key', 'dask', '_base')
+    _default_get = staticmethod(threaded.get)
+    _optimize = staticmethod(lambda dsk, keys, **kwargs: dsk)
+
+    @property
+    def _finalize(self):
+        return partial(_finalize_estimator, base=self._base)
```

Comment:
It's a property that returns a partial containing a reference to the base class on which it needs to be assigned. There is no reference to the parent object here, it's effectively the same as a staticmethod.